gravitee-apim-console-webui/src/management/api/list/api-list.component.html

```diff
@@ -144,7 +144,7 @@ <h1>APIs</h1>
 
     <!-- Display Version Column -->
     <ng-container matColumnDef="definitionVersion">
-      <th mat-header-cell *matHeaderCellDef id="definitionVersion">Version</th>
+      <th mat-header-cell *matHeaderCellDef id="definitionVersion">Gravitee Version</th>
```

Comment:
in json definition, the field is indeed called `gravitee`
but I'm not sure it's very helpful to call it like that in the UI

what about "Mode" ? so we have a Policy Studio mode or a Path based one ?

Reply:
Ping @audrey-tlm 😁

Reply:
'Mode' sounds perfect :) works well too for future event-native/asynchronous

Reply:
✅